index.html

```diff
@@ -1677,19 +1673,15 @@
                 </li>
               </ol>
             </li>
-            <li>If the <a>presentation availability promise</a> for
-            <code>presentationRequest</code> is not <code>null</code>, return
-            the <code>presentationRequest</code> object's <a>presentation
-            availability promise</a> and abort these steps.
+            <li>If there is an unsettled <a>Promise</a> from a previous call to
```

Comment:
I'll have to check our implementation here to see how to match the spec.  We just landed a patch to make the Promise an internal slot of the Availability object so the same one is always returned.

https://codereview.chromium.org/2572473003